src/cpu/cpu_eltwise_list.cpp

```diff
@@ -38,7 +39,7 @@ namespace {
 using namespace dnnl::impl::data_type;
 
 // clang-format off
-const pd_create_f impl_list[] = {
```

Comment:
Hi @kawakami-k 
I was wondering about the need for this change especially because `impl_list` is defined within anonymous namespace.

Reply:
Thank you the comment.
Sorry, it's a just copy miss. It seems that the source code from the past commit got mixed up.
I'll fix it.

Reply:
It looks good, thanks!